### DISCUSSION TITLE
Unable to connect to any device since macOS 18 release:

### DISCUSSION THREAD

**Discussion by yomitron** [iterate-ch/cyberduck#16521] [Q&A] [answered]
Since adopting macOS 18 CyberDuck is unable to connect to any device via FTP on a network. (No offense, FileZilla has no issues but your product is far superior!).

I reverted back to macOS 17 and have had no issues. The error in the screenshot happens to all bookmarked IPs and any new IP addresses added. 

<img width="258" alt="Screenshot 2024-11-08 at 3 17 08 PM" src="https://github.com/user-attachments/assets/47662e57-acd5-4181-b998-d1752f95731f">

See attached screenshot.

**Comment by AliveDevil** [comment]
Can you verify that Cyberduck has permission to access local network?

![image](https://github.com/user-attachments/assets/f7ac0312-c110-416b-bb4d-b1e0e5058394)

**Comment by TownerFiltration** [comment]
I'm having the same issue. I'm on macOS Sequoia 15.1.1 and latest version of Cyberduck. Cyberduck has permission to access local network. I installed Crossover and installed WinSCP and am able to connect to sftp and ftp locations. After WinSCP connects, then Cyberduck usually connects to sftp.

**Comment by dkocher** [comment, marked as answer]
Please try if changing the following settings resolves the problem:

* Disable firewall in _Settings → Network → Firewall_
* Disable any active network filter in _System Settings → Network → Filters_
* Set _System Settings → Wi-Fi → Details → Private Wi-Fi address → Fixed_

**Comment by yomitron** [comment]
Yup. This fixes it. Back on Cyberduck and happy as heck ^_^

**Comment by TownerFiltration** [comment]
No. Firewall is not enabled on my computer.

**Comment by dkocher** [comment]
Can you check if there are any network filters enabled on the system visible in _System Settings → Network → Filters_.

**Comment by TownerFiltration** [comment]
The System Settings → Network doesn't have Filters listed at all. Would they be in something else?

**Comment by TownerFiltration** [comment]
Here is a follow up for anyone having this problem. macOS Sequoia added a Private Wi-Fi address function. I noticed it while trying to remote into the computer trying the IP address that was assigned to the computer and finding out it had a different IP address. I turned off Private Wi-Fi address and Cyberduck started working fine. Next day and it is still working fine. Hoping it stays that way.
